.github/workflows/zig-fmt.yml

```
@@ -0,0 +1,17 @@
+name: Zig fmt
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  zig-fmt:
+    name: Run `zig fmt`
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+    - name: Install Zig
+      run: ./.github/bin/install-zig.sh
+
+    - name: Check that Zig code is formatted
+      run: |
+        "${HOME}/bin/zig" fmt --check .
```

Could the previous step install zig into a location that is already on the path? Then this could become:

```suggestion
        zig fmt --check .
```

---

Let's take this as another issue. I think it might turn into a rabbit hole